icekit/publishing/models.py

```diff
@@ -533,6 +533,19 @@ def clone_relations(self, dst_obj):
                 # re-created on publish thus always have empty M2M rels.
                 dst_m2m.add(*src_m2m.all())
 
+    def is_suppressed_message(self):
+        """
+        Occasionally items may not be visible to the public even if they have
+        been published and can be previewed. For example, if they belong to a
+        parent who is not published.
+
+        In that case, returning a string here will result in a red `published`
+        indicator in admin, and the string will be shown in the hyperlink
+        title.
+
+        :return: True always
+        """
+        return True
```

Comment:
The naming and return value of this method is confusing. `is_foo()` sounds like it should return a boolean. And it does, unless it's a string (which also evaluates truthy). This should be a property `suppressed_message` or a method `get_suppressed_message()` which always returns a string (an empty one when it's not suppressed, which evaluates falsey).